externs/ngeox.js

```diff
@@ -327,6 +327,39 @@ ngeox.MeasureEvent.prototype.feature;
 
 
 /**
+ * Options for the mobile geolocations directive.
+ * @typedef {{
+ *    accuracyFeatureStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    positionFeatureStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    zoom: (number|undefined)
+ * }}
+ */
+ngeox.DesktopGeolocationDirectiveOptions;
```

Comment:
Could'nt you have a single type for mobile and desktop or not ?
Do they differ, or could differ in the future ?